content/docs/1.7.1/deploy/upgrade/_index.md

```diff
@@ -16,7 +16,7 @@ Starting with v1.5.0, Longhorn only allows upgrades from supported versions. Whe
 Moreover, Longhorn does not support downgrades to earlier versions. This restriction helps prevent unexpected system behavior and issues associated with function incompatibility, deprecation, or removal.
 
 > **Warning**:
-> - Once you successfully upgrade to v1.7.0, you will not be allowed to revert to the previously installed version.
+> - Once you successfully upgrade to v1.7.1, you will not be allowed to revert to the previously installed version.
```

Comment:
Can we use the template variable for the current version?

Reply:
Updated. Thank you.